### PR TITLE
Created initial screen for pregame setup

### LINKED ISSUE #1134: Add initial screen for Pre-Game Setup
## Description
As a Developer,
I need to add initial screen for pregame setup,
so that the user has a choice to choose NPCs for the scenario conversations.

### Update [Required]
- [ ] Create xml layout for the screen

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 1 mentor.

## Estimation
2 hours

### PR DESCRIPTION
**Pull request by Rimjhim28**
### Description
Added initial screen for PreGame Setup. 

Fixes #1134 

### Type of Change:

- Code
- User Interface

**Code/Quality Assurance Only**
- New feature (non-breaking change which adds functionality pre-approved by mentors)

### How Has This Been Tested?
screenshot_2018-05-17-20-58-07-582_powerup systers com powerup

### Checklist:

- [ ] My PR follows the style guidelines of this project
- [ ] I have performed a self-review of my own code or materials

**Code/Quality Assurance Only**
- [ ] My changes generate no new warnings

### CODE REVIEW

File: PowerUp/app/build.gradle [hunk 1]
```diff
@@ -22,12 +21,14 @@ android {
             proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
         }
     }
+    buildToolsVersion '27.0.3'
```
Comment (m-murad) [inline PowerUp/app/build.gradle]: Declare this in the project level build.gradle file and use it here as a variable

File: PowerUp/app/build.gradle [hunk 2]
```diff
@@ -22,12 +21,14 @@ android {
...
 }
 
 dependencies {
     implementation fileTree(dir: 'libs', include: ['*.jar'])
     testImplementation "junit:junit:$rootProject.jUnitVersion"
     implementation "com.android.support:appcompat-v7:$rootProject.supportLibraryVersion"
+    implementation 'com.android.support.constraint:constraint-layout:1.1.0'
```
Comment (m-murad) [inline PowerUp/app/build.gradle]: Declare version as a variable in project level build.gradle file

File: .travis.yml [hunk 3]
```diff
@@ -4,7 +4,7 @@ android:
  components:
     - platform-tools
     - tools
-    - build-tools-26.0.2
+    - build-tools-27.0.3
     - android-26
```
Comment (m-murad) [inline .travis.yml]: Update this aswell.

Comment (Rimjhim28) [reply .travis.yml]: @m-murad Changing android to 27 is causing the travis to fail. Is there anything else that needs to updated/added?

File: PowerUp/app/build.gradle (hunk 2 above)
Comment (Rimjhim28) [reply PowerUp/app/build.gradle]: constraint error
@m-murad Changing to variable is giving an error. Can you please help.

File: .travis.yml (hunk 3 above)
Comment (m-murad) [reply .travis.yml]: What is the error message?

**Pull request merged**